apps/docs/src/components/Downloads/slidedecks/pitch-deck-slides.ts

```
@@ -1,8 +1,15 @@
-import type { Slide } from "../SlideDeckDownload";
+import type { Slide, KeyPoint } from "../SlideDeckDownload";
```

ESLint

[error] 'KeyPoint' is defined but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)